taskcluster/ci/email/kind.yml

```diff
@@ -15,13 +15,19 @@ jobs:
     worker-type: succeed
     run-on-tasks-for: [github-release]
     dependencies:
+      build: build-release
```

Comment:
Nit: What does specifying this dependency add? As far as I understand `Sign for Github`, already depends on `build-release`, I don't think we need to depend on `build-release` explicitly.

https://github.com/mozilla-mobile/firefox-tv/blob/50e5e72e8e913750cd5d669c26002a8cd43a6678/taskcluster/ci/signing/kind.yml#L18

Reply:
Oh, but I have an artifact-reference to `<build/...>`. Doesn't that mean I need to depend on `build-release` explicitly?

Reply:
Oh, that's a good question. I'm not sure for this particular case. Anyway, let's keep it for now. `kind-dependencies` will make manual `dependencies` go away, anyway.